### ISSUE TITLE
question about deliveryMethodId

### ISSUE THREAD

**Issue by yangcen6** [allegro/allegro-api#9680]
If we use ship with allegro, we want to call **/shipment-management/shipments/create-commands** to create one shipment, where can we get **deliveryMethodId**? From **delivery.method.id** in /order/checkout-forms/{id}?

**Comment by MartaNowaczyk** [comment]
Yes, from checkout and from [GET /shipment-management/delivery-services](https://developer.allegro.pl/documentation/#operation/getDeliveryServices).

**Comment by yangcen6** [comment]
Can we get **deliveryMethodId** from **/shipment-management/shipments/create-commands** before call **/shipment-management/shipments/create-commands** to create a shipment for this order?

**Comment by MartaNowaczyk** [comment]
You provided the same two paths (/shipment-management/shipments/create-commands) so I don't know what you mean.

**Comment by yangcen6** [comment]
Sorry. Can we get deliveryMethodId from **/shipment-management/shipments/create-commands** before call **/order/checkout-forms/{id}** to create a shipment for this order?

**Comment by MartaNowaczyk** [comment]
1. You can use [GET /shipment-management/delivery-services](https://developer.allegro.pl/documentation/#operation/getDeliveryServices) to retrieve a list of delivery services available to the authorized user. 
2. The method of delivery that the buyer has chosen you will see in the order /order/checkout-forms/{id} 
3. To create a new package, use [POST /shipment-management/shipments/create-commands](https://developer.allegro.pl/documentation/#operation/createNewShipment)

We described everything step by step in our [guide](https://developer.allegro.pl/tutorials/how-to-manage-parcels-via-ship-with-allegro-ZM9YAyGKWTV) (to change language settings to English please use the top-right icon).

**Comment by yangcen6** [comment]
For an order that has already created shipment using [POST /shipment-management/shipments/create-commands](https://developer.allegro.pl/documentation/#operation/createNewShipment), can we call this again?

**Comment by MartaNowaczyk** [comment]
Yes, you will create another shipment.
